packages/core/src/oidc/grants/client-credentials.ts

```diff
@@ -61,8 +65,40 @@ export const buildHandler: (
 
   const dPoP = await dpopValidate(ctx);
 
-  // eslint-disable-next-line @typescript-eslint/no-empty-function
-  await checkResource(ctx, async () => {});
+  /* === RFC 0001 === */
```

Comment:
Do we have the organization-level machine-to-machine feature included in the RFC0001?

Reply:
no, i think i should write a new one

Reply:
will update later